src/v/pandaproxy/schema_registry/store.h

```diff
@@ -112,6 +112,37 @@ class store {
         return res;
     }
 
+    ///\brief Get the global compatibility level.
+    result<compatibility_level> get_compatibility() const {
+        return _compatibility;
```

Comment:
> it looks like this would always return a valid enum, so why result? maybe this an interface whose implementation will change in a later patch?

It keeps the public interface consistent.